tests/mmm/test_delayed_saturated_mmm.py

```diff
@@ -710,12 +714,6 @@ def test_new_data_predict_method(
 
     assert isinstance(posterior_predictive_mean, np.ndarray)
     assert posterior_predictive_mean.shape[0] == new_dates.size
-    # Original scale constraint
-    assert np.all(posterior_predictive_mean >= 0)
-
-    # Domain kept close
-    lower, upper = np.quantile(a=posterior_predictive_mean, q=[0.025, 0.975], axis=0)
-    assert lower < toy_y.mean() < upper
```

Comment:
Does it make sense for these tests not to work with the changes? They seem like rather useful checks

Reply:
Commented this above. I will add these to the tests from the actual fit. 
Using the normal likelihood doesn't guarantee the prior will meet these constraints. They were geared toward a fit model. Not a prior predictive